app/views/result/index.html.erb

```diff
@@ -59,61 +59,77 @@
+
+                <a href = "<%= side.link %>" >
+                  <%= truncate(side.name, :length => 60 ) %>
                 </a>
+
                 <p><%= side['title'] %></p><br>
+
                 <span class="head-info">
-                  <%= head_info_list_maker(side).compact.join(", ") %>
+                  <% head_fields = Searching::HEAD_FIELDS.select { |field| enabled?(field) } %>
+                  <%= side.get_head_info_list(head_fields).compact.join(", ") %>
+                  <%=
+                    render(
+                      :partial => "content_id",
+                      :locals => {
+                        :side => side
+                      }
+                    )
+                  %>
+                  <% second_fields = Searching::SECONDARY_HEAD_FIELDS.select { |field| enabled?(field) } %>
+                  <%=
+                    render(
+                      :partial => "people_and_organisations",
+                      :locals => {
+                        :side => side,
+                        :fields => second_fields
+                      }
+                    )
+                  %>
                 </span>
+
                 <br>
                 <span class="head-info-2">
                   <%= truncate(side['description'], :length => 200) %>
                 </span>
+
+
+                <% enhanced_fields = Searching::ENHANCED_FIELDS.select { |field| enabled?(field) } %>
+                <%=
+                  render(
+                    :partial => "enhanced_results_table",
+                    :locals => {
+                      :enhanced_results => side.enhanced_results(enhanced_fields)
+                    }
+                  )
+                %>
+
               </p>
             </th>
           <% end %>
 
           <% score_difference = results.score_difference(results.right[i]['link'], i) %>
-          <% if score_difference == "0" %>
-            <th>
-              <div class="changeless-box">
-                <p class="table-body">
-                  N/A
-                </p>
-              </div>
-            </th>
-          <% elsif score_difference.start_with?("-") %>
-            <th>
-              <div class="down-box">
-                <p class="table-body">
-                  <%= score_difference %>
-                </p>
-              </div>
-            </th>
-          <% else %>
-            <th>
-              <div class="up-box">
-                <p class="table-body">
-                  <%= "+#{score_difference}" %>
-                </p>
-              </div>
-            </th>
-          <% end %>
-
+          <%=
+            render(
+              :partial => "up_down_box",
+              :locals => {
+                :score_difference => score_difference
+              }
+            )
+          %>
         </tr>
       <% end %>
 
       <tr class="table-row">
-        <th></th>
+        <th/>
```

Comment:
Why did you do this? while it is fine for some tags, I am not 100% sure `th`'s are included in that list.

I would suggest it is clearer in the long form here.

Reply:
👍